lib/auth/server_info_test.go

```diff
@@ -94,18 +94,25 @@ func TestReconcileLabels(t *testing.T) {
 	require.NoError(t, err)
 
 	// Update the server's labels.
-	labels := map[string]string{"a": "1", "b": "2"}
-	serverInfo, err := types.NewServerInfo(types.Metadata{
-		Name:   "aws-my-account-my-instance",
-		Labels: labels,
+	awsServerInfo, err := types.NewServerInfo(types.Metadata{
+		Name:   types.ServerInfoNameFromAWS("my-account", "my-instance"),
+		Labels: map[string]string{"a": "1", "b": "2"},
 	}, types.ServerInfoSpecV1{})
 	require.NoError(t, err)
-	serverInfo.SetSubKind(types.SubKindCloudInfo)
-	require.NoError(t, pack.a.UpsertServerInfo(ctx, serverInfo))
+	awsServerInfo.SetSubKind(types.SubKindCloudInfo)
+	require.NoError(t, pack.a.UpsertServerInfo(ctx, awsServerInfo))
+
+	regularServerInfo, err := types.NewServerInfo(types.Metadata{
+		Name:   types.ServerInfoNameFromNodeName(serverName),
```

Comment:
Sorry for the silly Q, but what makes awsServerInfo and regularServerInfo be reconciled together?

Reply:
All ServerInfos that match a node are reconciled at the same time, with the order determined by `getServerInfoNames()`.